gcc/rust/ast/rust-ast.cc

```diff
@@ -1814,14 +1814,13 @@ IfLetExpr::as_string () const
   str += append_attributes (outer_attrs, OUTER);
 
   str += "\n Condition match arm patterns: ";
-  if (match_arm_patterns.empty ())
+  if (match_arm_pattern == nullptr)
```

Comment:
This looks wrong, checking for null should be avoided checking for empty is much safer.